sql/data_model.sql

```diff
@@ -0,0 +1,64 @@
+BEGIN;
+
+-- add pgcrypto extension so that we can use the gen_random_uuid function.
+CREATE EXTENSION IF NOT EXISTS pgcrypto;
+
+DROP TABLE IF EXISTS formdata CASCADE;
+CREATE TABLE formdata(
+  id uuid PRIMARY KEY DEFAULT gen_random_uuid(),
+  domain text NOT NULL,
+  received_on timestamp without time zone,
+  time_start timestamp without time zone,
+  time_end timestamp without time zone,
+  duration bigint,
+  device_id text,
+  user_id uuid NOT NULL,
+  username text,
+  app_id uuid,
+  xmlns text NOT NULL,
+  form_json text NOT NULL
+);
+
+CREATE INDEX formdata_domain on formdata(domain);
+
+DROP TABLE IF EXISTS casedata CASCADE;
+CREATE TABLE casedata(
+  id uuid PRIMARY KEY DEFAULT gen_random_uuid(),
+  domain text NOT NULL,
+  version text,
+  type text,
+  closed boolean DEFAULT FALSE,
+  user_id uuid,
```

Comment:
:+1:  on `last_modifier_id`

Agree that many of the fields aren't necessary for the endpoint and I don't think we really want to do analytics on this DB. I'll remove them for now since it will make it easier to run the tests.